scripts/feature_extractor_contextualized.py

```diff
@@ -0,0 +1,149 @@
+import torch
+from transformers import BertTokenizer, BertModel
+
+
+class BertExtractor:
+
+    def __init__(self, bert_model, max_len, lower_case):
+        """
+        This class contains methods to extract contextualized word embeddings with a given Bert model.
+        :param bert_model: which of the pretrained Bert model should be loaded.
+        :param max_len: this pads the sentences to a maximum length such that computing the embeddings can be done for
+        batches and is more efficient
+        :param lower_case: whether to lower case or not
+        """
+        self._model = BertModel.from_pretrained(bert_model, output_hidden_states=True)
+        self._model.eval()
+        self._tokenizer = BertTokenizer.from_pretrained(bert_model, do_lower_case=lower_case)
+        self._max_len = max_len
+
+    def convert_sentence_to_indices(self, sentence):
+        """
+
+        :param sentence: A String representing a sentence, without special tokens
+        :param max_len: an int : the length to which the sentence will be padded
```

Comment:
max_len is not a parameter in this method but an instance variable. so probably not needed here